apm-sniffer/apm-toolkit-activation/apm-toolkit-log4j-2.x-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/log/log4j/v2/x/TraceIdConverterActivation.java

```diff
@@ -21,45 +21,37 @@
 
 import net.bytebuddy.description.method.MethodDescription;
 import net.bytebuddy.matcher.ElementMatcher;
-import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.ClassStaticMethodsEnhancePluginDefine;
-import org.apache.skywalking.apm.agent.core.plugin.match.NameMatch;
-import org.apache.skywalking.apm.agent.core.plugin.interceptor.StaticMethodsInterceptPoint;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.ConstructorInterceptPoint;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.InstanceMethodsInterceptPoint;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.ClassInstanceMethodsEnhancePluginDefine;
 import org.apache.skywalking.apm.agent.core.plugin.match.ClassMatch;
 
 import static net.bytebuddy.matcher.ElementMatchers.named;
+import static org.apache.skywalking.apm.agent.core.plugin.match.NameMatch.byName;
+
 
 /**
- * Active the toolkit class "org.apache.skywalking.apm.toolkit.log.logback.v2.x.LogbackPatternConverter".
- * Should not dependency or import any class in "skywalking-toolkit-logback-2.x" module.
- * Activation's classloader is diff from "org.apache.skywalking.apm.toolkit.log.logback.v2.x.LogbackPatternConverter",
- * using direct will trigger classloader issue.
- *
- * @author wusheng
+ * @author xuhe
  */
-public class Log4j2OutputAppenderActivation extends ClassStaticMethodsEnhancePluginDefine {
 
-    public static final String ENHANCE_CLASS = "org.apache.skywalking.apm.toolkit.log.log4j.v2.x.Log4j2OutputAppender";
-    public static final String INTERCEPT_CLASS = "org.apache.skywalking.apm.toolkit.activation.log.log4j.v2.x.PrintTraceIdInterceptor";
-    public static final String ENHANCE_METHOD = "append";
+public class TraceIdConverterActivation extends ClassInstanceMethodsEnhancePluginDefine {
```

Comment:
Can you rename class `TraceIdConverterActivation` to make it end with `Instrumentation`, something like `TraceIdConverterInstrumentation`, I know the original class doesn't end with this, but we now use the suffix to verify the `import`s.

Reply:
In the toolkit, it should be named w/ `Activation` suffix. We may need to fix the test check.

Reply:
Well, I'll adjust the check soon

Reply:
Maybe bootstrap-plugin also need to check.